cmd/server/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+	migrate "github.com/rubenv/sql-migrate"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/FHA-FB5/FortuneTeller/pkg/api"
+	"github.com/FHA-FB5/FortuneTeller/pkg/database"
+	"github.com/FHA-FB5/FortuneTeller/pkg/migrations"
+)
+
+func main() {
+	var ipAddress, dbAddress string
+	var port int
+	flag.StringVar(&ipAddress, "ip", "0.0.0.0", "ip address to listen on")
+	flag.IntVar(&port, "port", 5000, "port to listen on")
+	flag.StringVar(&dbAddress, "db", "postgres://postgres@postgres:5432?sslmode=disable",
+		"address of the postgres database")
+	flag.Parse()
+	ctx := context.Background()
+	logger := log.StandardLogger()
+	var db *sqlx.DB
+	var err error
+	for i := 0; i < 5; i++ {
```

Comment:
don't you think it would be pretty good to set the retries by flag?

Reply:
sure